using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -500,16 +570,20 @@ ACPIdentity.getUrlVariables {(urlVariables) in
 
 ### getUrlVariables <a id="geturlvariables-js"></a>
 
-_added in react-native-acpcore v1.0.5_
+_{% hint style="info" %}
+This method was added in react-native-acpcore v1.0.5.
+{% endhint %}_
 
-Retrieve Adobe visitor data as a URL query parameter string for consumption in hybrid mobile applications. There is no leading "?" or "&" punctuation, as the caller is responsible for placing the string in the correct location of their resulting URL. The following information is added to the string that is returned via the callback:
+This `getUrlVariables()` API gets the Visitor ID Service variables in URL query parameter form for consumption in hybrid app. This method will return an appropriately formed string containing Visitor ID Service URL variables. There will be no leading (&) or (?) punctuation, as the caller is responsible for placing it in their resulting java.net.URI in the correct location.
```

Comment:
- Please change to "This API gets the...."
- Please reword to say:
"This API gets the Visitor ID Service variables in URL query parameter form, and these variables will be consused in the hybrid app. This method returns an appropriately formed string that contains the Visitor ID Service URL variables. There will be no leading (&) or (?) punctuation because the caller is responsible for placing the variables in their resulting java.net.URI in the correct location."